cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -190,6 +190,15 @@ func (o *AddOrUpdateOptions) Run(args []string) error {
 		return err
 	}
 
+	if o.URL == existingRepository.Spec.Fetch.ImgpkgBundle.Image {
+		repoWatcher := NewRepoTailer(o.NamespaceFlags.Name, o.Name, o.ui, client, RepoTailerOpts{
+			PrintCurrentState: true,
+		})
+		err = repoWatcher.TailRepoStatus()
+		return err
+
```

Comment:
nit: Let's do it inline
```suggestion
		return NewRepoTailer(o.NamespaceFlags.Name, o.Name, o.ui, client, RepoTailerOpts{PrintCurrentState: true}).TailRepoStatus()
```

Reply:
fixed